freva-rest/Dockerfile

```diff
@@ -61,8 +60,8 @@ ENV API_CONFIG=/opt/freva-rest/api_config.toml \
     SOLR_PORT=8983 \
     MONGO_PORT=27017 \
     # usr pass
-    MONGO_USER=mongo \
-    MONGO_PASSWORD=secret \
+    MONGO_USER="" \
+    MONGO_PASSWORD="" \
```

Comment:
But now they are empty strings by default. What I meant is that you shouldn't assign default values (set this) at all. The same goes for the DB names env variable.